engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/videoSettings/ScreenshotFormatBinding.java

```diff
@@ -29,12 +31,12 @@ public ScreenshotFormatBinding(RenderingConfig config) {
     @Override
     public ScreenshotFormat get() {
         switch (config.getScreenshotFormat()) {
-        case "png":
-            return ScreenshotFormat.PNG;
-        case "jpeg":
-            return ScreenshotFormat.JPEG;
+        case PNG:
```

Comment:
This seems a bit odd.... why not just check return the format unless null, and return the default value in that case?